Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -1581,6 +1588,20 @@ static void webkit_settings_class_init(WebKitSettingsClass* klass)
         FALSE,
         readWriteConstructParamFlags);
 
+    /**
+     * WebKitSettings:enable-websecurity:
+     *
+     * Enable or disable support for Web Security on pages.
```

Comment:
This needs a comment here saying what exactly gets disabled (cross-domain checks, CORS, CSP, maybe others?) and a big fat scary warning telling people that they **will be hacked** if they navigate the open Internet with this setting disabled.

Reply:
I added some additional comment and warning but any suggestions are welcome